Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "one"
+version = "0.1.0"
+authors = ["athulya"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = "~2.27.0"
+chrono = "0.4.10"
```

Comment:
add a new line at the end.....each file should have a new line at the end....this is a common code practice followed globally